frontend/management/src/components/ApplicantsTable.tsx

```diff
@@ -0,0 +1,42 @@
+import { ApplicantInterface } from "../utils/types";
+
+function ApplicantsTable({
+  applicants,
+  goToApplicantPage,
+}: ApplicantsTableProps) {
+  return (
+    <>
+      <table className="table">
+        <thead>
+          <tr>
+            <th scope="col">Name</th>
+            <th scope="col">Email</th>
+            <th scope="col">Status</th>
+            <th scope="col">Score</th>
+          </tr>
+        </thead>
+        {applicants.map((applicant) => {
+          return (
+            <tbody>
+              <tr>
+                <th scope="row" onClick={() => goToApplicantPage(applicant.id)}>
```

Comment:
Is there a reason that it is a th tag?

Reply:
We use it to have a "leading" column in the table